src/Registry/TypeRegistry.php

```diff
@@ -337,6 +338,11 @@ public function init_type_registry( TypeRegistry $type_registry ) {
 		PostObjectUnion::register_type( $this );
 		TermObjectUnion::register_type( $this );
 
+		/**
+		 * Register scalar.
+		 */
+		Upload::register_type();
```

Comment:
We should probably register scalars at the top of this method, to allow any of the types preceding it.

Reply:
I'm assuming stanning is failing because Composer's autoloader hasnt been updated. `composer dump-autoload --no-dev --optimize` should fix that.